bffnt.py

```diff
@@ -255,22 +257,21 @@ def load(self, json_filename):
         }
 
         for entry in range(cmap['start'], cmap['end'] + 1):
-            utf16 = unichr(entry)
+            try:  #Python2
+                utf16 = unichr(entry)
+            except:  #Python3
+                utf16 = chr(entry)
             if utf16 in glyph_map:
                 cmap['entries'][utf16] = glyph_map[utf16]
 
         self.cmap_sections = [cmap]
 
-    def _int_sort(self, x, y):
-        x = int(x, 10)
-        y = int(y, 10)
-        if x < y:
-            return -1
-        if x > y:
-            return 1
-        return 0
+    def _int_sort(self, n):
```

Comment:
This isn't a sorting function anymore, just a conversion from string to base 10.